app/views/events/_transfer_form.html.erb

```diff
@@ -6,30 +6,41 @@
     data-transfer-form-wire-value="<%= new_event_wire_path(@event) %>">
     <div data-transfer-form-target="home">
       <fieldset>
-        <legend class="heading h2 pt2 pb2">What type of transfer?</legend>
+        <legend class="heading h2 pb2">What type of transfer?</legend>
         <div class="field field--options fit">
           <%= link_to new_event_ach_transfer_path(@event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
             <%= inline_icon "payment-transfer", size: 28 %>
             <strong>ACH transfer</strong>
+            <small>United States</small>
           <% end %>
           <%= link_to new_event_increase_check_path(@event), class: "text-decoration-none btn--form-option", data: { turbo_frame: "_top" } do %>
             <%= inline_icon "email", size: 28 %>
             <strong>Mailed check</strong>
+            <small>United States</small>
```

Comment:
nit: Domestic?

Reply:
I think we should design our copy for US-based organisations, all orgs are technically operating in the US. We could do a conditional.